Unwrap.xcodeproj/project.pbxproj

```diff
@@ -3004,15 +3005,16 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = "iMessage App Icon";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 2;
-				DEVELOPMENT_TEAM = B5C26XE59E;
+				DEVELOPMENT_TEAM = 6389FT2N69;
```

Comment:
Pls, do not change the bundle identifier and the development team. 🙂

Reply:
How do I run locally and also not change the development team?
It won't compile unless I change the team.